policyengine_us/parameters/gov/states/mn/tax/income/subtractions/pension_income/cap.yaml

```diff
@@ -22,7 +22,7 @@ JOINT:
         interval: 10
 HEAD_OF_HOUSEHOLD:
   values:
-    2023-01-01: 12_250
```

Comment:
was it 12_250 in 2022 / 2021?

Reply:
No, this pension income subtraction was added in 2023. 
![image](https://github.com/user-attachments/assets/86b95f7f-95ca-46d8-b192-2a25b236bbed)